src/ledger.js

```diff
@@ -183,9 +195,9 @@ export class LedgerInteraction extends DirectKeystoreInteraction {
    * }
    */
   async withApp(callback) {
```

Comment:
If the awaits are removed, does this still need to be async?

Reply:
Yes - I believe you're right here. I removed the async in the new PR I submitted. (closing this one). 

https://github.com/unchained-capital/unchained-wallets/pull/24/files#diff-5da93726c6b0fe6dfa4d1b76cdc3767dL185 but you have to open the diff on ledger.js